babybear/src/main/java/frc/robot/control/InstalledHardware.java

```diff
@@ -0,0 +1,58 @@
+// ***********************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Charged Up - 2023
+// File: InstalledHardware.java
+// Intent: Forms a listing of switches that will help to debug code better as hardware is available (or not available).
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+
+package frc.robot.control;
+
+/**
+ * A class devoted to installed hardware constants. Use this to decide if
+ * hardware is enabled on the robot or not.
+ * All developers to use this to protect their subsystems and commands from
+ * using hardware that is not actually installed
+ * on the robot at hand. Used to assist in development stages and make it easier
+ * to quickly remove a piece of hardware
+ * from the robot.
+ */
+public class InstalledHardware {
+    // Installed I/O Electronic hardware
+    // Hardware that is N/A - mostly because these pieces of hardware have no
+    // communication with the RoboRio
+    // batteryInstalled = true;
+    // voltageRegulartorModuleInstalled = true;
+    // robotEnabledLightInstalled = true;
+    // roboRioInstalled = true; - if it wasn't installed then the code wouldn't be
+    // running
+
+    // Basic Hardware
+    public static final boolean powerDistributionPanelInstalled = true;
+
+    // Onboard Hardware - Orentation/Navigation Hardware
+    public static final boolean wifiRadioInstalled = true;
+    public static final boolean navx2Installed = false;
+    public static final boolean navx1Installed = true;
+    public static final boolean navxInstalled = navx1Installed || navx2Installed;
```

Comment:
Conversation has been around only using navx2 for the fall robot.

Reply:
Changed to say navx2 installed